src/pages/index.js

```diff
@@ -321,36 +321,29 @@ const IndexPage = () => (
     <div className={styles.genericBanner}>Not sure where to stay?</div>
 
     <div className={styles.sectionSubheader}>
-      Try <a href="https://www.marriott.com/hotels/travel/nycal-aloft-manhattan-downtown-financial-district/">Aloft Manhattan Downtown</a>
+      Some recommended hotels near the venue: <a href="https://www.marriott.com/hotels/travel/nycal-aloft-manhattan-downtown-financial-district/">Aloft Manhattan Downtown</a>,
```

Comment:
```suggestion
      Some recommended hotels near the venue: <a href="https://www.marriott.com/hotels/travel/nycal-aloft-manhattan-downtown-financial-district/">Aloft Manhattan Downtown</a>,{" "}
```
This adds a space between commas.